src/rttui/channel.rs

```diff
@@ -23,7 +24,10 @@ pub struct ChannelState {
     down_channel: Option<DownChannel>,
     name: String,
     format: DataFormat,
+    /// contains Strings when [ChannelState::format] is [DataFormat::String]
     messages: Vec<String>,
+    /// when [ChannelState::format] is not [DataFormat::String]
+    /// contains rtt binary data or binary data which is Defmt format
     data: Vec<u8>,
```

Comment:
Just as a side note, these two should probably be converted to an enum (`enum Data { Binary(Vec<u8>), Text(Vec<String>) }`) in the future, right? Doesn't need to be in this PR though.

Reply:
Hm I think the wording was not very good on my part, if I recall correctly that should mean "If the channel format is not String, then this Vec contains binary data. This data is either raw rtt data or it is data received via rtt but in defmt format and in this case needs further processing to be converted to Strings. If the channel format is String, this Vec is not used".